### ISSUE TITLE
Supplement color-coding of charts with pattern

### ISSUE THREAD

**Issue by GitGators** [tl-its-umich-edu/my-learning-analytics#1250]
Location: Resources Accessed - personal views vs class views - better contrast between the blue and gray area

**Comment by jennlove-um** [comment]
Add a pattern to the bars on the resources accessed.  Work with the ITS Accessibility team, especially Gonzalo Silverio for suggestions.

**Comment by jaydonkrooss** [comment]
Created a branch https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1553 with my initial changes.
Here's what I've proposed, using patterns that ought to contrast with the original colors of the bars:

![Screenshot 2023-11-14 at 12 13 24 PM](https://github.com/tl-its-umich-edu/my-learning-analytics/assets/147423601/94585378-83a6-4c24-a06d-ae8d85554b8f)

I've only edited the resources accessed visualization and nothing else, so let me know if we may need to think about other pages of MyLA to improve or update with this change.

**Comment by jaydonkrooss** [comment]
@jennlove-um That's a fair point, I could see that the solid bar would have enough of a contrasting pattern. I wasn't given explicit design ideas from Accessibility, I was just going off of an example I saw from a blog post originating from one of the resources they gave me: https://keen.io/blog/accessibility-in-data-vis/
I'll bring the solid blue bar back, and if needed I could also try to get the accessibility team's explicit approval of either option.

**Comment by jennlove-um** [comment]
Testing fails for this issue.  With the numbers pushed outside the bars, the numbers are overlapping with the brush slider on the side. 

<img width="581" alt="image" src="https://github.com/tl-its-umich-edu/my-learning-analytics/assets/39634416/e7f5cf16-c37e-40b5-9a74-5f78a75b1bd7">

We need to revisit this to avoid that overlap.

**Comment by jennlove-um** [comment]
Testing passes in beta
